doc/build/sysbuild/index.rst

```diff
@@ -386,6 +393,23 @@ specify this file when building with sysbuild, as follows:
          :gen-args: -DAPP_DIR=samples/hello_world -DSB_CONF_FILE=sysbuild-mcuboot.conf
          :compact:
 
+Sysbuild targets
+****************
+
+Sysbuild creates build targets for each image (including sysbuild itself) for
+the following modes:
+
+ * menuconfig
+ * hardenconfig
+ * guiconfig
+
+For the main application (as is the same without using sysbuild) these can be
+ran normally without any prefix. For other images (including sysbuild), these
+are ran with a prefix of the image name and an underscore e.g. ``sysbuild_`` or
+``mcuboot_``, using ninja or make. Note that flash and debug modes are not
+included when using sysbuild, flashing and debugging should be done using west
+commands instead.
```

Comment:
Sysbuild flashing and debugging using `west` has already been described earlier in this document:
https://docs.zephyrproject.org/latest/build/sysbuild/index.html#sysbuild-flashing-using-west-flash

so I don't think it is needed again to state that `west` must be used for flashing and debugging.

Also the fact that those targets are not part of the list x_config` list above should hint at this. 

But not a blocker for me, so if you really think it should be here, i'm ok with that.

Reply:
It's more to present an example of things that are not covered e.g. tfm*, *_report, etc.

Reply:
> It's more to present an example of things that are not covered e.g. tfm*, *_report, etc.

Thanks for the reasoning.
I think that a subsection describing how such targets can be invoked would make a lot of sense.
Either as part of this PR or a dedicated one. 

For example:
```
Dedicated image build targets
===================

a good description including something like .....
Dedicated image build targets which are not mapped by sysbuild targets, such as `tfm*`, `ram_report`,
`rom_report`, etc can be invoked in the image's build folder, using `ninja` or `west`.

For example when building hello_world using `west`, then rom_report for hello_world can be generated as:
Build hello_world:
west build -b reel_board --sysbuild samples/hello_world

Generating rom_report using `west`:
west build -d build/hello_world -t rom_report

Generating rom_report using `ninja`:
ninja -C build/hello_world rom_report